test/e2e/kubectl/kubectl.go

```diff
@@ -1072,7 +1072,9 @@ metadata:
 			framework.ExpectNotEqual(schema, nil, "retrieving a schema for the crd")
 
 			meta := fmt.Sprintf(metaPattern, crd.Crd.Spec.Names.Kind, crd.Crd.Spec.Group, crd.Crd.Spec.Versions[0].Name, "test-cr")
-			validArbitraryCR := fmt.Sprintf(`{%s,"spec":{"bars":[{"name":"test-bar"}],"extraProperty":"arbitrary-value"}}`, meta)
```

Comment:
this was a false-negative where client-side validation didn't know how to validate any properties in a custom resource that included x-kubernetes-preserve-unknown-fields anywhere. the adjusted fixture is an unknown field that both client-side and server-side validation will complain about